nifi-commons/nifi-properties/src/main/java/org/apache/nifi/util/NiFiProperties.java

```diff
@@ -321,10 +322,13 @@
     // analytics defaults
     public static final String DEFAULT_ANALYTICS_PREDICTION_ENABLED = "false";
     public static final String DEFAULT_ANALYTICS_PREDICTION_INTERVAL = "3 mins";
```

Comment:
There was an issue at the time of development where if the value for the DEFAULT_ANALYTICS_QUERY_INTERVAL and DEFAULT_ANALYTICS_PREDICTION_INTERVAL were identical then we were seeing invalid computations due to a difference  in the way some threads were accessing the data. A fix was to increase the query interval to be larger than the prediction interval. This should not affect the computations since they are based upon the prediction interval. It would perhaps delay the first computation from being calculated by two minutes, after that there would be no effect. This problem may be corrected with NIFI-6801. If so, we can revert the value back to 3 at a later time.

Reply:
Sounds good. Thanks!